_scripts/api-reference.sh

```diff
@@ -2,21 +2,33 @@
 
 set -e
 
-API_VERSION="$(cat .api-version)"
+API_VERSION="develop"
```

Comment:
@EkimChau heads up we'll need to make sure this change is reverted when we release 6.0, otherwise it'll be building the incorrect API docs.